File: תרגיל_1/backend/DAL/models/Hmomembers.cs
using System;
using System.Collections.Generic;

namespace DAL.models
{
    public partial class Hmomembers
    {
        public Hmomembers()
        {
            CoronaPatients = new HashSet<CoronaPatients>();
            VaccineToMember = new HashSet<VaccineToMember>();
        }

        public int Id { get; set; }
        public string Identity { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime? BirthDate { get; set; }
        public string Telephone { get; set; }
        public string Phone { get; set; }
        public string Address { get; set; }
        public string ProfileImage { get; set; }

        public virtual ICollection<CoronaPatients> CoronaPatients { get; set; }
        public virtual ICollection<VaccineToMember> VaccineToMember { get; set; }
    }
}

File: תרגיל_1/src/app/classes/HMOMembers.ts
import { CoronaPatients } from "./CoronaPatients";
import { VaccineToMember } from "./VaccineToMember";

export class HMOMembers{
    constructor(public Id?: Number, public Identity?: Number, public FirstName?: String, public LastName?: String,
        public BirthDate?: Date, public Telephone?: String, public Phone?: String, public Address?: String,
        public ProfileImage?: String, public CoronaPatients?: Array<CoronaPatients>, public VaccineToMember?: Array<VaccineToMember>) 
    { }
}